### ISSUE TITLE
[java] InvalidLogMessageFormat flags logging calls using a slf4j-Marker

### ISSUE THREAD

**Issue by kris-scheibe** [pmd/pmd#2250]
**Affects PMD Version:** 6.21.0

**Rule:** [InvalidLogMessageFormat](https://pmd.github.io/latest/pmd_rules_java_errorprone.html#invalidlogmessageformat)

**Description:**

InvalidLogMessageFormat interprets the marker as the log message and can't determine the correct number of arguments.

Before #2196 those log messages were probably just ignored...

```java
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.slf4j.Marker;
import org.slf4j.helpers.BasicMarkerFactory;

public class InvalidLogMessageFormatTest {
    private static final Logger logger = LoggerFactory.getLogger("MyLogger");
    private static final Marker marker = BasicMarkerFactory.getMarker("MARKER");

    public static void main(String[] args) {
        logger.warn(marker, "foo {}", "bar"); // gets flagged
    }
}
```

**Running PMD through:** *Other*
